mlrun/runtimes/base.py

```diff
@@ -599,6 +599,9 @@ def _force_handler(self, handler):
         if not handler:
             raise RunError(f"handler must be provided for {self.kind} runtime")
 
+    def _has_pipeline_param(self) -> bool:
+        return self.to_json().find("{{pipelineparam:op") > 0
```

Comment:
please add a comment that points to pipeline_param regex for more info/full regex (I was mainly confused by why the brackets are not closed)

Reply:
re-used the regex pipelineparam and added reference to it in the regex file